README.md

```diff
@@ -1,73 +1,86 @@
+> [!IMPORTANT]
+> **Degree Planner is in alpha.**\
+> Degree Planner is not feature complete, and not yet ready for widespread adoption. However, it is important to the contributing community that we develop openly and transparently. If you’re interested in helping us build Degree Planner, we do recommend you use it. Then, check out the [open issues](https://github.com/SOFTENG310-G7/degree-planner/issues), and perhaps [open a new one](https://github.com/SOFTENG310-G7/degree-planner/issues/new/choose) if you see fit!
 
-### Technology Stack
+Planned features include:
 
-This is a web-based project using React, Next.js and TypeScript, Tailwind CSS in conjunction with Supabase.
+- save and share degree plans as well as rating courses based on past experiences;
+- incorporate university entrance requirements; and
+- recommended course plans tailored for specific occupations.
 
-### How You Can Contribute
+## 🍔 Technology stack
 
-Please take a look at the [contributing guidelines](CONTRIBUTING.md) and [code of conduct](CODE_OF_CONDUCT.md)
+Degree Planner is a React-based web app build with [Next.js](https://nextjs.org), [TypeScript](https://www.typescriptlang.org) and [Tailwind CSS](https://tailwindcss.com), backed by a [Supabase](https://supabase.com) data store.
 
-### Local Repository Deployment
+> [!NOTE]
+> Degree Planner is tested with the latest **long-term support** version of  Node.js, version&nbsp;18. Though it is likely to work with newer releases, they are not officially supported.
 
-**Step 1**
-  ```
-  git clone https://github.com/SOFTENG310-G7/degree-planner.git
-  ```
+## 🫵 How you can contribute
 
-**Step 2**
-  Install Node.js version 18 LTS.
+Thank you for your interest in contributing to degree planner! There are plenty of ways to help us build Degree Planner, only part of which is has to involve writing code. You could help us by providing feedback, reporting bugs, writing documentation, testing, helping with design, or suggesting features and changes you’d like to see.
 
-**Step 3**
-  ```
-  npm install
-  ```
+It can be daunting to contribute to a project you aren’t familiar with. Simply using Degree Planner is a good place to start, and if you’ve got some ideas to contribute, check out our [contributing guidelines](/CONTRIBUTING.md) and [code of conduct](/CODE_OF_CONDUCT.md). Happy contributing!
 
-**Step 4**: Create an .env.local file in the root directory and insert with your specific settings:
+## 🚀 Deploying locally
 
-  ```
-  NEXT_PUBLIC_SUPABASE_URL=
-  NEXT_PUBLIC_SUPABASE_ANON_KEY=
-  SUPABASE_SERVICE_KEY=
+### Step 1
 
-  UOA_CLIENT_ID=
-  UOA_CLIENT_SECRET=
-  ```
+```sh
+git clone https://github.com/SOFTENG310-G7/degree-planner.git
+```
 
-  - The first three can be obtained from the properties of your Supabase instance. The last two will have to be generated [here](https://developer.auckland.ac.nz/prd/dashboard).
+### Step 2
 
-**Step 5**
-  ```
-  npm run dev
-  ```
+Install Node.js version 18 LTS.
```

Comment:
We should also link this to the Node.js website

Reply:
I agree, but do you think we should do it in this PR? Note from above:

> The _Local deployment_ instructions are still a bit sparse on detail, but I think that warrants its own issue and PR.

If we’re fleshing that whole section out (which I think is necessary), it might make more sense to do it in that context. I can open a new issue for it.

Reply:
Yeah I think if we are doing another PR for the instructions this can be done there. Happy for this to be merged.